epictrack-web/src/components/project/ProjectList.tsx

```diff
@@ -50,6 +52,21 @@ const ProjectList = () => {
 
   const projects = React.useMemo(() => ctx.data as Project[], [ctx.data]);
 
+  async function fetchProponents() {
+    const project_proponents = projects
+      .map((p) => p.proponent_id)

```

Comment:
let's try using full words for variables please for any new implementation

Reply:
sounds good